src/tiledb/cloud/bioimg/ingestion.py

```diff
@@ -340,7 +343,8 @@ def register_dataset_udf(
         _SUPPORTED_EXTENSIONS,
         *args,
         verbose=verbose,
-        access_credentials_name=acn,
```

Comment:
This is breaking the main ingest. We list `acn` in the ingest function as an arg, but don't pass it into our first node. So this throws an access denied error if an access credential error is passed into `acn` rather than `access_credentials_name` @ktsitsi @ihnorton @Shelnutt2